src/tests/ftest/dfuse/container_attrs.py

```diff
@@ -0,0 +1,113 @@
+"""
+  (C) Copyright 2020-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import re
+
+from apricot import TestWithServers
+from dfuse_utils import get_dfuse, start_dfuse
+
+
+class DfuseContainerAttrs(TestWithServers):
+    """Check if the dfuse attributes of a container are properly managed.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        # obtain separate logs
+        self.update_log_file_names()
+
+        # Start the servers and agents
+        super().setUp()
+
+    def _check_attrs(self, dfuse, container_name):
+        """Check if the DFuse attributes of a container are loaded
+
+        Check in the log file of the dfuse instance if it contains the DFuse attributes of a given
+        container.  It also checks the value of the attributes found.
+
+        Args:
+            dfuse (Dfuse): DFuse instance to check
+            container_name (str): Name of the container
+        """
+        log_file = os.linesep.join(dfuse.get_log_file_data().output[0].stdout)
+        attrs = self.params.get("attrs", f"/run/{container_name}/*")
+        for name, value in [attr.split(':') for attr in attrs.split(",")]:
+            match = re.findall(
+                fr"^.+\ssetting\s+'{name}'\s+is\s+(\d+)\s+seconds$",
+                log_file,
+                re.MULTILINE)
+            self.assertEqual(
+                len(match),
+                1,
+                f"Unexpected number setting(s) of attribute {name}: want=1, got={len(match)}")
+            self.assertEqual(
+                value,
+                match[0],
+                f"Unexpected value for attribute {name}: want={value}, got={match[0]}")
+
+    def test_dfuse_container_attrs(self):
+        """Jira ID: DAOS-14698.
+
+        Test Description:
+            Create a container with DFuse attributes
+            Mount a DFuse mount point
+            Check the output of the DFuse log
+        :avocado: tags=all,pr
```

Comment:
Checking - do these tests really need to be `pr`? Or could they be `daily_regression` or `full_regression`?

Reply:
I have no opinion on this, and do not fill legitimate to have one.
I will let you decide with the other test members.

Reply:
We need your input :) Roughly, it's something like
- Is the test critical and quick?
  - yes -> `pr`
  - no -> Is the test fairly important **_or_** very quick?
    - yes -> `daily_regression`
    - no -> `weekly_regression`

IMO these new tests belong in `daily_regression` unless they are very critical to base funcionality. @ashleypittman thoughts?

Related, I think we should add the `daos_cmd` feature tag to these tests since they verify some GO code paths for the `daos` command

Reply:
- [ ] Add  `daos_cmd` feature tag
- [ ] Change tags from `pr` to `daily_regression`

Reply:
> * [x]  Add  `daos_cmd` feature tag
> * [x]  Change tags from `pr` to `daily_regression`

Fixed with commit 7f267334e